OneCommunityGlobal/HighestGoodNetworkApp
# Zuhang_fix_compressed_view_of_dashboard_tasks_tab

# Description
- On the dashboard page, there are two columns: the left column(tasks and timelogs) and the right column(Leaderboard).
- Previously both columns shrink at the same time, making the information on the left not visible.
- Jae now want that when he shrink the browser window, only the right column shrinks and the left column stays the same. 

For more detail, please check the below link for what's needed:
https://www.loom.com/share/698ff0609a3c45879581b8557b10eb7c?sid=11912a5d-6c0f-4534-95bd-cd03a2dcb196

## Main changes explained:
- Update file Dashboard.jsx for the corresponding components. 
- Update file Dashboard.css. Creating new css styles for the components.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to dashboard
6. verify when you shrink the browser window, only the right columns shrinks. And the left stays the same until there's not enough spaces for any.

## Screenshots or videos of changes:
Before changes:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/123051008/6e82d2fe-5947-4c34-8c86-0e6a9238813d

After changes: 
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/123051008/18d63fd5-aa2c-473f-a4fd-ea9556abb847



## Note:
Include the information the reviewers need to know.
